sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -290,15 +309,24 @@ export class EventHubClient {
   }
 
   /**
-   * Creates an EventHubConsumer that can be used to receive events from the Event Hub for which this
-   * client was created.
+   * Creates an Event Hub consumer responsible for reading `EventData` from a specific Event Hub parition,
+   * and as a member of a specific consumer group.
+   *
+   * A consumer may be exclusive, which asserts ownership over the partition for the consumer
+   * group to ensure that only one consumer from that group is reading the from the partition.
+   * These exclusive consumers are sometimes referred to as "Epoch Consumers."
+   *
+   * A consumer may also be non-exclusive, allowing multiple consumers from the same consumer
+   * group to be actively reading events from the partition.  These non-exclusive consumers are
+   * sometimes referred to as "Non-epoch Consumers."
```

Comment:
I don't see the benefit of talking about non-exclusive consumers here.

Reply:
I see benefit in discussing the two types, especially given that we've collapsed two dedicated methods for creating the specific types into one single one.   In my opinion, this lays out the potential options available and invites developers to look more closely at the options class itself if they're interested in more context.

Reply:
I think it could be helpful to discuss the two types here since this is the only method that a customer can use to create a consumer, and it's an important consideration when creating a consumer.

Reply:
Even if we do want to talk about the 2 types, the most common case should be spoken of first which in this case is the "non-exclusive" one.  **if**/**when** one wants to have exclusive ownership, **then** they use the fancy option. How about the below?

> Multiple consumers are allowed on the same partition in a consumer group. If there is a need to have an exclusive consumer for a partition in a consumer group, then specify the `ownerLevel` in the options. Such exclusive consumers were previously referred to as "Epoch Receivers"